subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ val tomljVersion = "1.0.0"
 val archunitVersion = "1.2.0"
 val bytebuddyVersion = "1.10.20"
 val jettyVersion = "9.4.36.v20210114"
-val sshdVersion = "2.0.0" // Upgrade requires changes in package names and tests fail on expectations (but work otherwise)
+val sshdVersion = "2.10.0"
```

Comment:
🤔 the latest version is `2.12.0` any reason not to update further?

Reply:
Above `2.10`, the whole API was redesigned, and it's a much bigger work to get the expectations going vs. `2.10.0`, which is relatively the same as `2.0.0`

Reply:
We can see about updating further, but that version is driven by the upcoming JGit upgrade.